### DISCUSSION TITLE
How to solve a model with internal walls in Baram?

### DISCUSSION THREAD

**Discussion by akshaymani94** [nextfoam/baram#122] [Q&A]
I have a .unv mesh file that has internal walls and baffles to simulate porous wall and fan boundary conditions. I had done the meshing in Salome. When I import the mesh in baramFlow it does not detect the internal boundaries. Is there any way around this?

**Comment by jiban** [comment]
As you may know, BARAM utilizes `ideasUnvToFoam` OpenFOAM utility to convert `.unv` mesh to OpenFOAM polyMesh.
The problem you mentioned comes from the utility itself...
We could do it if we find a customer that will fund the development because it is not a trivial job.
I'm sorry for that and will try to find one.
Thank you for bringing the needs up.

**Reply by akshaymani94** [reply]
Thankyou SIr for the response. I thought Baram had the feature of CreateBafflesDict to specify the boundary conditions for the internal faces. If not could suggest me an alternative way to solve this issue.

**Comment by kdg5424** [comment]
Hello @akshaymani94. This is Donggyu from NEXTFOAM.

There is a issue with mesh if you generate with salome.
I'm not sure right now, but I guess if the mesh is generated like prism in salome, the mesh would not be converted into polyMesh.
So could you check it?

Thanks
